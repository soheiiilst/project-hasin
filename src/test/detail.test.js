import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import store from '../redux/store';
import DetailPage from '../pages/detail/detail.component';

it('expect to render DetailPage component', () => {
  expect(
    shallow(
      <Provider store={store}>
        <DetailPage />
      </Provider>
    )
  ).toMatchSnapshot();
});
