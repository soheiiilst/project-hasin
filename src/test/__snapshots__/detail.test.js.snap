// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render DetailPage component 1`] = `ShallowWrapper {}`;
